src/main/java/de/atextor/turtle/formatter/TurtleFormatter.java

```diff
@@ -121,6 +112,24 @@ public String apply( final Model model ) {
         return outputStream.toString();
     }
 
+    public String applyToContent( final String content ) {
```

Comment:
This method should have a JavaDoc, that specifically clarifies that the content String is supposed to contain a valid RDF/Turtle document.

Reply:
added comments to all 3 public methods